examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx

```diff
@@ -0,0 +1,321 @@
+import { Frame, Json, OnlineUserInfo } from "@mirohq/websdk-types";
+import {
+  useBreakout,
+  useOnlineUsers,
+  useSelectedItems,
+  useTimer,
+} from "../../hooks";
+import { convertTime, formatDisplayTime, isUser } from "../../utils";
+import { RoomConfig } from "../RoomConfig/RoomConfig";
+import { DEFAULT_TIME, Timer } from "../Timer/Timer";
+import {
+  DropdownMenu,
+  IconButton,
+  IconPlus,
+  IconHandFilled,
+  IconDotsThreeVertical,
+  Button,
+} from "@mirohq/design-system";
+
+import "./BreakoutManager.css";
+import { BreakoutStarter } from "../BreakoutStarter";
+import { WaitingList } from "../WaitingList";
+
+export const BreakoutManager: React.FC = () => {
+  const { breakout, rooms, isFacilitator, ...service } = useBreakout();
+  const onlineUsers = useOnlineUsers();
+  const selectedItems = useSelectedItems<Frame>({
+    predicate: (item) => item.type === "frame",
+  });
+  const [selectedRoom, setSelectedRoom] = React.useState<Room>();
+  const [duration, setTimerDuration] = React.useState<number>();
+  const [currentTime, setCurrentTime] = React.useState<number>(0);
+  const [canUseTimer] = React.useState<boolean>(false);
+
+  const onTimerStop = React.useCallback(() => {
+    service.endSession();
+  }, [breakout?.id]);
+
+  const timer = useTimer({
+    duration: duration ?? DEFAULT_TIME,
+    onStop: onTimerStop,
+    onTick: (timestamp) => setCurrentTime(timestamp),
+  });
+
+  const participantIds = rooms
+    .map((room) => room.participants.map((p) => p.id))
+    .flat()
+    .sort()
+    .join("-");
+
+  const unassignedUsers = React.useMemo(() => {
+    return onlineUsers.filter((user) =>
+      rooms.every((room) =>
+        room.participants.every((participant) => participant.id !== user.id),
+      ),
+    );
+  }, [onlineUsers, participantIds]);
+
+  React.useEffect(() => {
+    const handleSelectionUpdate = async () => {
+      if (!selectedItems.length || !selectedRoom) {
+        return;
+      }
+
+      const [frame] = selectedItems;
+
+      if (frame) {
+        await service.setRoomTarget(selectedRoom, frame.id);
+        await miro.board.notifications.showInfo(
+          `Frame "${frame.title}" has been selected as starting point.`,
+        );
+        await miro.board.deselect({ id: frame.id });
+        setSelectedRoom(undefined);
+      } else {
+        await miro.board.notifications.showError(
+          "We only support frames as starting point for now",
+        );
+      }
+    };
+
+    handleSelectionUpdate();
+  }, [selectedRoom, selectedItems]);
+
+  React.useEffect(() => {
+    const boostrap = () => {
+      const handleNudge = async (currentUser?: Json) => {
+        if (isUser(currentUser)) {
+          await miro.board.notifications.showInfo(
+            `<strong>${currentUser?.name}</strong> is waiting to start the session`,
+          );
+        }
+      };
+
+      miro.board.experimental.events.on("nudge-facilitator", handleNudge);
```

Comment:
This should not work anymore. I moved the events out of the experimental namespace

Reply:
It didn't before, let me rebase with the main branch and fix this.

Reply:
Done